### PR TITLE
Adding new required env variable of MMDB_LICENSE_KEY and removing thr…

### PR DESCRIPTION
**Pull request by aaroncouch**
MaxMind recently restricted access to their GeoLite2 databases. It is now required that anyone wanting to access their databases must create an account with them. More information can be found [here](https://blog.maxmind.com/2019/12/18/significant-changes-to-accessing-and-using-geolite2-databases/)

I also noticed an issue with Scouter itself. The ICMP ping utility was rarely returning negative floats for the round-trip-times. I believe this issue is due to us utilizing threading to execute tests in parallel and the very slim chance that packets are being created with duplicate IDs. I've swapped out threading with multiprocessing and all usages of RandShort() in the ping utility have been replaced with os.getpid() as to guarantee that all packets are sent with a unique identifier.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ You'll need docker installed in order to run this container.
 * [macOS](https://docs.docker.com/mac/started/)
 * [Linux](https://docs.docker.com/linux/started/)
 
+You'll also need to [sign up for a GeoLite2 account](https://www.maxmind.com/en/geolite2/signup), and generate a license key. License keys can be generated free of charge and are used to authorize access to download MaxMind databases.
```
Comment (klaude) [inline README.md]: There's an extra comma before "and generate a license key".

Comment (klaude) [inline README.md]: Can you add a quick sentence saying what Scouter needs the MaxMind database for?

Comment (aaroncouch) [reply README.md]: Fixing now.

Comment (aaroncouch) [reply README.md]: Sure can.

**Pull request merged**